e2e/e2e-test-setup.js

```diff
@@ -0,0 +1,26 @@
+import { execSync } from 'child_process'
+import * as path from 'path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+const appDir = path.join(__dirname, 'test-app')
+
+execSync('cargo build --release', { stdio: 'inherit' })
```

Comment:
The `--relase` flag just improve the built binary but does not have any impact. To speed up the build process, I'd do: 

```suggestion
execSync('cargo build -C opt-level=0', { stdio: 'inherit' })
```

Reply:
You mean `cargo build --config opt-level=0` ? Because `-C` flag should stand for cwd